cmake/packaging/windows.cmake

```
@@ -103,7 +103,7 @@ set(CPACK_NSIS_DELETE_ICONS_EXTRA
 # Checking for previous installed versions
 set(CPACK_NSIS_ENABLE_UNINSTALL_BEFORE_INSTALL "ON")
 
-set(CPACK_NSIS_HELP_LINK "https://sunshinestream.readthedocs.io/about/installation.html")
+set(CPACK_NSIS_HELP_LINK "https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/installation.html")
```

```suggestion
set(CPACK_NSIS_HELP_LINK "https://sunshinestream.readthedocs.io/en/latest/about/installation.html")
```

I've been mostly using the original readthedocs url, in case anything happens to our domain, things "should" still work. RTD used to automatically resolve to `/en/latest/` ... not sure why they don't anymore.